static/scss/app.scss

```diff
@@ -0,0 +1,20 @@
+// Protocol
+// @import "libs/protocol/protocol";
```

Comment:
As part of #848 – we want to move tokens/vars into the Protocol design system. We will do this later. Leaving commented out for now.